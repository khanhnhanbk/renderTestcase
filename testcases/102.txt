"A string with an empty \n line"