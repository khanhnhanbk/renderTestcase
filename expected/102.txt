A string with an empty \n line,<EOF>